lib/client.dart

```diff
@@ -1,166 +1,3 @@
-import 'dart:async';
+import 'interface.dart';
```

Comment:
This is a breaking change for the package. Please document it in the `CHANGELOG.md`.

Reply:
Also: what's the role of this file?

Reply:
Yes, it was until I made a new update and it's now working well.

Reply:
It is role is to act as an indirect access point to the whole [Client](https://github.com/isoos/pwa/blob/master/lib/client.dart#L6) class.